### ISSUE TITLE
Migrate experiment specific `did` columns to json based generic metadata

### ISSUE THREAD

**Issue by maxnoe** [rucio/rucio#6843]
### Description

Looking at the output of `rucio get-metadata` and then going over these definitions here:

https://github.com/rucio/rucio/blob/ba102506d470c417fd2b136304e4fa4f7fc3a870/lib/rucio/db/sqla/models.py#L431-L453

there seem to be a lot of columns for did metadata that seem very experiment specific and not general for all user communities of rucio.

E.g. SKAO or CTAO won't have anything like a `lumiblocknr` 

It is of course only slightly annoying to have fields we will never use clutter the output, but maybe for one of the next major versions, this could be cleaned up?

### Steps to reproduce

See above

**Comment by bari12** [comment]
Yes, this is something annoying and purely exists for historic reasons. It's something I want to change together with @Geogouz but we need to be quite careful about it. Ideally, I would like to migrate these VO-specific columns (run_number, stream_name, etc.) to the generic json column in a transparent way, and then drop the columns completely from the table.

**Comment by rdimaio** [comment]
Related discussion: https://github.com/rucio/rucio/discussions/6576

**Comment by Geogouz** [comment]
Added @bari12 as a reviewer to discuss the direction and planning.

**Comment by bari12** [comment]
Thank you for raising this. We have reviewed this issue as part of our regular issue review.

This issue has been open for longer than the twelve-month delivery horizon, but we are making an exception and keeping it open. 

Will be worked on by @Geogouz within the next 12m

In line with our [issue management guideline](https://rucio.cern/documentation/developer/issue_management), such exceptions are granted case by case, and no issue remains open for longer than twenty-four months. By then this issue will be resolved, re-justified, or closed and labelled `wishlist`.
